### PR TITLE
Fix DIAL additionaldataUrl port settings

### PR DESCRIPTION
**Pull request by Damian-Danylko**

### CODE REVIEW

File: DIALServer/DIALServer.cpp [hunk 1]
```diff
@@ -286,7 +286,6 @@ namespace Plugin {
             _dialURL = Core::URL(service->Accessor());
             _dialURL.Host(selectedNode.HostAddress());
             _dialPath = '/' + _dialURL.Path().Value();
-            _webServerPort = _dialURL.Port();
```
Comment (pwielders) [inline DIALServer/DIALServer.cpp:289]: Tgis looks scary, now the _webServerPort is not set. PLease double check if this is really good.

File: DIALServer/DIALServer.cpp [hunk 2]
```diff
@@ -380,7 +379,7 @@ namespace Plugin {
         } else {
             // FIXME: At the moment part of additionalDataUrl parameter is hardcoded, localhost is obligatory by Netflix 
             // but rest of the path can be created dynamically or should be retrived from configuration    
-            const string additionalDataUrl = (_T("http://localhost") + ((_webServerPort != 80)? _T("") : _T(":") + Core::NumberType<uint16_t>(_webServerPort).Text()) + _T("/Service/DIALServer/Apps/") + app.Name() + _T("/") + _DefaultDataExtension);
+            const string additionalDataUrl = (_T("http://localhost") + ((_webServerPort == 80)? _T("") : _T(":") + Core::NumberType<uint16_t>(_webServerPort).Text()) + _T("/Service/DIALServer/Apps/") + app.Name() + _T("/") + _DefaultDataExtension);
```
Comment (pwielders) [inline DIALServer/DIALServer.cpp:382]: This I understand and indeed is a bug. Good Fix

**Review by pwielders — CHANGES_REQUESTED** [review]
Please double check the removal of the _webServerPort. I think it is an accidental remove and it should be in

File: DIALServer/DIALServer.cpp (hunk 1 above)
Comment (Damian-Danylko) [reply DIALServer/DIALServer.cpp]: I forgot to commint header file. Now implementation should cover both case when WebServer is off and on. In both cases 
_webServerPort will be set properly.

**Review by pwielders — APPROVED** [review]

**Pull request merged**